orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/StageDefinitionBuilder.java

```diff
@@ -40,24 +46,62 @@ default void taskGraph(
     @Nonnull Stage stage, @Nonnull Builder builder) {
   }
 
-  default @Nonnull List<Stage> aroundStages(
-    @Nonnull Stage stage) {
+  @Deprecated
+  default @Nonnull List<Stage> aroundStages(@Nonnull Stage stage) {
     return emptyList();
   }
 
+  @Deprecated
   default @Nonnull List<Stage> parallelStages(
     @Nonnull Stage stage) {
     return emptyList();
   }
 
-  default @Nonnull List<Stage> afterStages(@Nonnull Stage stage) {
-    return emptyList();
+  default void beforeStages(
```

Comment:
Default implementation uses `aroundStages` — I'd like to remove this ultimately but this means I didn't have to change all the stages right away.